src/clusterfuzz/_internal/bot/tasks/utasks/symbolize_task.py

```diff
@@ -105,8 +105,10 @@ def utask_main(uworker_input):
   else:
     build_revision = testcase.crash_revision
 
+  fuzz_target = testcase_manager.get_fuzz_target_from_input(uworker_input)
```

Comment:
I think this probably is unneeded because symbolize task is never done on libfuzzer targets.